assets/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "amethyst-assets"
+version = "0.1.0"
+authors = ["torkleyy <torkleyy@gmail.com>"]
+readme = "README.md"
+description = """
+Asynchronous asset management for games.
+"""
+
+[dependencies]
+fnv = "1"
+futures = "0.1.14"
+parking_lot = "0.4.4"
```

Comment:
I basically agree with what @Aceeri said: I prefer to just specify the latest patch version, since I don't know whether or not I'm using features of a patch, so it could work for me and on CI, but the dependency version isn't correct.